website/docs/plugins/joblib_launcher.md

```diff
@@ -58,21 +49,20 @@ hydra:
     temp_folder: null
 
     # thresholds size of arrays that triggers automated memmapping
-    max_nbytes: 1M
+    max_nbytes: null
 
     # memmapping mode for numpy arrays passed to workers
     mmap_mode: r
 ```
 
-`n_jobs` defaults to -1 (all available CPUs) and `prefer` defaults to `processes`. Depending on the application, `threads` might be a good alternative. All arguments specified in `joblib` are passed to `Joblib.Parallel` (see [`Joblib.Parallel` documentation](https://joblib.readthedocs.io/en/latest/parallel.html) for full details). 
+`n_jobs` defaults to -1 (all available CPUs). See [`Joblib.Parallel` documentation](https://joblib.readthedocs.io/en/latest/parallel.html) for full details on arguments. Note that the backend is hard-coded to use process-based parallelism (Joblib's loky backend), since thread-based parallelism is incompatible with Hydra's logging.
```

Comment:
```suggestion
`n_jobs` defaults to -1 (all available CPUs). See [`Joblib.Parallel` documentation](https://joblib.readthedocs.io/en/latest/parallel.html) for full details on arguments. Note that the backend is hard-coded to use process-based parallelism (Joblib's loky backend), since thread-based parallelism is incompatible with Hydra.
```